### PR TITLE
Add effects row to profile list

### PR DESCRIPTION
**Pull request by edlerd**

### CODE REVIEW

File: static/js/ProfileList.tsx [hunk 1]
```diff
@@ -38,6 +45,44 @@ const ProfileList: FC = () => {
...
+        {touchesCloudInit && (
+          <Tooltip message="Cloud init" position="btm-center">
+            <Icon name="restart" />
+          </Tooltip>
+        )}
+        {touchesLimits && (
+          <Tooltip message="Limit resources" position="btm-center">
+            <Icon name="priority-low" />
+          </Tooltip>
+        )}
```
Comment (lorumic) [inline static/js/ProfileList.tsx:82]: The icons look a bit too close to each other - should we set some (even little) spacing between them?

File: static/js/ProfileList.tsx [hunk 2]
```diff
@@ -38,6 +45,44 @@ const ProfileList: FC = () => {
...
+      (device) => device.type === "nic"
+    );
+    const touchesStorage = Object.values(profile.devices).some(
+      (device) => device.type === "disk"
+    );
+    const touchesCloudInit = Object.keys(profile.config).some((config) =>
+      config.startsWith("cloud-init")
+    );
+    const touchesLimits = Object.keys(profile.config).some((config) =>
+      config.startsWith("limits")
```
[the remark this answers is not in the document]
Comment (edlerd) [reply static/js/ProfileList.tsx:58]: The API only returns sections that are not-empty.

File: static/js/ProfileList.tsx (hunk 1 above)
Comment (edlerd) [reply static/js/ProfileList.tsx]: Yeah, I added an inline List as a wrapper to add some spacing.

**Review by lorumic — APPROVED** [review]
Very nice, LGTM!

**Pull request merged**